src/prompts/index.ts

```diff
@@ -94,6 +94,11 @@ export async function processPrompt(
   );
 
   if (!maybeFilePath(prompt.raw)) {
```

Comment:
@mldangelo Should I put the function checking inside the maybeFilePath branch? or even before that?

Reply:
Yes! That's a great idea if you don't mind making the change. Thank you!

Reply:
@mldangelo Amended and it passed all the unit tests